### ISSUE TITLE
Optimization issue

### ISSUE THREAD

**Issue by safi-alt** [AnusBaig/UI-templates#1]
The Following Template have lot of optimization issues and npm martket problems

**Comment by AnusBaig** [comment]
Yes it has too much. Do you have any better alternative that provides all dashboard related components?

**Comment by safi-alt** [comment]
Build your own from scratch or use components from this project and optimize according to your needs

**Comment by AnusBaig** [comment]
Templates are used to save time. Who builds from sratch
